src/hardware/sblaster.cpp

```diff
@@ -226,23 +237,49 @@ static int E2_incr_table[4][9] = {
 #define min(a,b) ((a)<(b)?(a):(b))
 #endif
 
+static const char * CardType() {
+	constexpr std::array<const char *, 8> types = {"None",   "SB1",
+	                                               "SBPro1", "SB2",
+	                                               "SBPro2", "Unasigned",
+	                                               "SB16",   "GB"};
```

Comment:
Place opening brace in a new line, please.

"Unasigned" -> "Unassigned" ?

Also, since these values sometimes are used for LOG_MSG prefix, maybe they should be all uppercase?

*Ah, I thought, that maybe std::make_array can be used, but it's not available in C++14… :(*

Reply:
All great catches; thank you!